src/test/kotlin/it/pagopa/aca/service/AcaServiceTests.kt

```diff
@@ -29,12 +32,14 @@ class AcaServiceTests {
     private val ibansClient: IbansClient = mock()
     private val acaUtils = AcaUtils()
     private val acaService = AcaService(gpdClient, ibansClient, acaUtils)
+    @Captor private lateinit var paymentPositionModelDtoCaptor: ArgumentCaptor<PaymentPositionModelDto>
```

Comment:
```suggestion
    private val paymentPositionModelDtoCaptor: KArgumentCaptor<PaymentPositionModelDto> =
        argumentCaptor<PaymentPositionModelDto>()
```
🚲 use kotlin argument captor here